org.jacoco.build/pom.xml

```diff
@@ -94,12 +94,12 @@
     <module>../org.jacoco.agent.rt</module>
     <module>../org.jacoco.agent</module>
     <module>../org.jacoco.ant</module>
-
+    <module>../org.jacoco.cli</module>
+    <module>../org.jacoco.examples</module>
```

Comment:
@Godin Yes, I think it was somehow confusing before: As `org.jacoco.tests` depends indirecty on `org.jacoco.examples` it is built before `org.jacoco.tests` anyways. Wanted to reflect the actual execution order here.